### ISSUE TITLE
Rails 6 installation

### ISSUE THREAD

**Issue by no1-knows** [DavyJonesLocker/client_side_validations#766]
Hi!
Can u let me know how to use new rails 6 app?
Especially where should I write `//= require rails.validations` ?

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the CSV Community.

I'm working on this, but it will require time because I'm still not satisfied of the result and at the moment I'm having trouble in understanding some stuff (I'm waiting for a clarification at turbolinks/turbolinks#481).

Take a look at the `move-to-es` branch if you are interested

Anyway...

To use CSV 15.0 with webpacker you can do the following:

### 1. Add jQuery to webpacker
Run
```sh
$ yarn add jquery
```

Edit `config/webpack/environment.js`
```js
const { environment } = require('@rails/webpacker')

const webpack = require('webpack')

// Add an additional plugin of your choosing : ProvidePlugin
environment.plugins.prepend(
  'Provide',
  new webpack.ProvidePlugin({
    // jQuery
    $: 'jquery',
    jQuery: 'jquery',
    jquery: 'jquery',

    // Window
    'window.jQuery': 'jquery',
  })
)

// export the updated config
module.exports = environment
```

### 2. Copy CSV's JavaScript asset
Run
```sh
$ rails g client_side_validations:copy_assets
```

Move `rails.validations.js` from `app/assets/javascripts` to `app/javascript`

### 3. Require CSV's Javascript file
Add the following line to `app/javascript/application.js` **after** `require("turbolinks").start()`

```js
require("rails.validations")
```

Hope it helps

Things will be better when I will release a dedicated node package.

**Comment by tagliala** [comment]
@no1-knows I've released 16.0.0 with a dedicated npm package to help integration with webpacker. 

Readme has been updated with new instructions (you still need to load jQuery via webpack, eg: https://github.com/diowa/ruby2-rails5-bootstrap-heroku/blob/master/config/webpack/environment.js#L19-L24)

Everything is working fine for me, feel free to give a try and report feedback
